composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "ezsystems/privacy-cookie-bundle",
+    "description": "Privacy cookie banner integration bundle into eZ Publish/eZ Platform",
+    "keywords": ["cookies", "privacy", "policy"],
+    "license": "GPL-2.0",
+    "authors": [
+        {
+          "name": "eZ Publish dev-team & eZ Community",
+          "homepage": "https://github.com/ezsystems/ezpublish-community/contributors"
+        }
+    ],
+    "require": {
+        "symfony/symfony": "~2.5",
```

Comment:
This bundle more like depends on Twig version than Symfony, so... I guess we can reduce requirements.